src/main/webapp/display-event.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Capstone</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="body">
+    <div class="header">
+      <div class="header-left">
+        <a href="/index.html">        
+          <img src="images/uniteLogo.svg" alt="Unite by STEP logo.">
+        </a>
+        <a href="/index.html">Home</a>
+        <a href="/create-event.html">Create</a>
+        <a href="/search.html">Find</a>
+      </div>
+      <div class="header-right">
+        <a href="/my-events.html">My Events</a>
+      </div>
+    </div>
+    <div class="event-display-container">
+      <div class="event-left-details">
+        <div class="event-display-image"></div>
+        <div class = "event-left-text">
+          <div class="event-display-header">
+            <div class = "event-display-title">Beach Clean Up</div>
+            <div class="attendee-count-container">
+                    <span class="attendee-count green">12</span> already attending</div>
```

Comment:
The new tag system I created would have this class be "attendee-count environment-text".
The "[tag]-text" class is used to change the color of the text to match what would normally be the tag's background color.

Reply:
Fixed now!